services/api/config/application.yml.example

```diff
@@ -18,25 +18,35 @@ production:
   secret_token: ~
   blob_signing_key: ~
   sso_app_secret: ~
-  sso_app_id: ~
-  sso_provider_url: ~
-  workbench_address: ~
-  websocket_address: ~
+  sso_app_id: arvados-server
+  sso_provider_url: https://uuid_prefix.your.domain
+  workbench_address: https://workbench.uuid_prefix.your.domain
+  websocket_address: wss://ws.uuid_prefix.your.domain/websocket
   #git_repositories_dir: ~
   #git_internal_dir: ~
+  #git_repo_ssh_base: "git@git.uuid_prefix.your.domain:"
+  #git_repo_http_base: https://git.uuid_prefix.your.domain/
+  #auto_admin_first_user: false
+  #crunch_job_wrapper: :slurm_immediate
+  #sso_insecure: false
```

Comment:
[Install docs](http://doc.arvados.org/install/install-api-server.html#omniauth) do say you need to add these *if* you aren't using the default/recommended locations for some reason. Normally, the defaults are fine and you don't need to do anything here.

Mentioning `auto_admin_first_user` in the install docs seems like a good idea -- it's surely more convenient if your API server isn't publicly accessible. The method given in the install doc (using rails console) is safer in that you get a chance to confirm you're giving root to the correct account before you actually do it (even though the docs don't show the part of the transcript where rails console displays the user details).

I do like adding `sso_insecure` here, right beside the other `sso_*` configs.